### ISSUE TITLE
Block "More" button: inconsistent ellipsis orientation

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#23889]
**Describe the bug**
In WordPress 5.5 the block toolbar has been slightly redesigned. The "More" button has now a horizontal ellipsis which is inconsistent with the top bar "More" button vertical ellipsis.

These two "More" buttons have similar functionality and meaning: their presentation should be similar to allow users to immediately identify their purpose.

As a user, looking at these two buttons, I'd be a bit confused:
- why these two buttons have a different ellipsis orientation?
- what the difference between the two is?
- what does a horizontal (or vertical) ellipsis mean in the first place?

The two ellipsis buttons in WordPress 5.4: they are consistent:

<img width="1280" alt="block more button vertical ellipsis" src="https://user-images.githubusercontent.com/1682452/87246540-57ef1180-c44e-11ea-958b-97fa2c131e59.png">

The two ellipsis buttons in WordPress 5.5: they are not consistent:

<img width="1280" alt="block more button horizontal ellipsis" src="https://user-images.githubusercontent.com/1682452/87246549-663d2d80-c44e-11ea-9222-295ac7fc36d9.png">

Apart from accessibility concerns and potential confusion for all users, this seems to me lack of basic UI consistency that should be fixed.

**Comment by pablohoneyhoney** [comment]
Fair observation. If everyone agrees, I think it’s ok to align and have the ellipsis be vertical in both cases for the time being. 

There was an intent to improve the icons within the toolbar, and horizontal felt an opportunity to improve the unity and balance of information within, while the top bar hasn’t been revisited or optimized yet. In the future, it could be worth challenging the complexity of the top right corner UI, as it causes semantic and functional confusion, especially between settings, ellipsis, and the proliferation of other functionalities –like other plugins or global styles.

**Comment by shaunandrews** [comment]
I think making them both vertical is fine. I'll remove the **Needs Design Feedback** label.

>  it could be worth challenging the complexity of the top right corner UI, as it causes semantic and functional confusion, especially between settings, ellipsis, and the proliferation of other functionalities –like other plugins or global styles.

Couldn't agree more.
